src/main/java/com/bringframework/factory/AutowiredBeanPostProcessor.java

```diff
@@ -0,0 +1,62 @@
+package com.bringframework.factory;
+
+import com.bringframework.exceptions.BeanInjectionException;
+import com.bringframework.registry.BeanDefinitionRegistry;
+import com.bringframework.resolver.DefaultDependencyResolver;
+import com.bringframework.resolver.DependencyResolver;
+import java.lang.reflect.Field;
+import java.util.Map;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * BeanPostProcessor that receives a map of raw bean instances, reads list of fields marked as
+ * autowired from {@link com.bringframework.registry.BeanDefinition} and injects into ther
+ * corresponding dependencies resolved by {@link DependencyResolver}
+ */
+
+@Slf4j
+public class AutowiredBeanPostProcessor implements BeanPostProcessor {
+  private final BeanDefinitionRegistry definitionRegistry;
+  private final Map<String, Object> rawBeans;
+  private final DependencyResolver dependencyResolver;
+
+  public AutowiredBeanPostProcessor(BeanDefinitionRegistry registry, Map<String, Object> rawBeans) {
+    this.rawBeans = rawBeans;
```

Comment:
Please follow the same style in your code, if you use "this" to set fields in constructor, make the same for all fields.

Reply:
fixed